app/src/main/java/com/hover/stax/home/StaxAppReview.kt

```diff
@@ -14,11 +14,7 @@ internal object StaxAppReview {
 
     fun launchStaxReview(activity: Activity) {
         AnalyticsUtil.logAnalyticsEvent(activity.getString(R.string.visited_rating_review_screen), activity)
-
-//        if (Utils.getBoolean(Constants.APP_RATED_NATIVELY, activity))
-//            openStaxPlaystorePage(activity)
-//        else
-//            launchReviewDialog(activity)
```

Comment:
I suggest best practice if you are not using this code, just remove it instead of commenting it out.

Reply:
Thanks! Done.